CGPs/0014.md

```diff
@@ -1,45 +1,62 @@
-# CGP [0014]:  cUSD Rewards
+# CGP [0014]:  Distribute CELO to cUSD holders
 
 - Date: 2020-10-25
-- Author(s): @ewilz, @cla-bel, @nambrot, @timmoreton
+- Author(s): @Jbsibille, @ewilz
 - Status: DRAFT
```

Comment:
I'd add this line:
`Discussions-To: https://forum.celo.org/t/governance-proposal-to-reward-early-users/662/26` to show folks where to go to discuss this CGP